0000-template.md

```diff
@@ -1,15 +1,22 @@
 # Feature name
 
 * Proposal: [SE-NNNN](NNNN-filename.md)
-* Author: [Swift Developer](https://github.com/swiftdev)
+* Authors: [Author 1](https://github.com/swiftdev), [Author 2](https://github.com/swiftdev)
```

Comment:
A single author is more common, so maybe that part of the template shouldn't change (to avoid d709546002e1636a10350d14da84eb9e554c3aac).